README.Rmd

```diff
@@ -81,32 +81,90 @@ estimate_static(
 )
 ```
 
-### Change in disease severity over the 1976 Ebola outbreak
+### Change in overall severity estimate over the 1976 Ebola outbreak
 
-Calculate the time-varying CFR estimates up to a given point in time.
+In this example we show how the severity estimate can change as more data on the outbreak becomes available, using the function `estimate_rolling()`.
```

Comment:
```suggestion
In this example we show how the estimate of overall severity can change as more data on cases and deaths over time becomes available, using the function `estimate_rolling()`. Because there is a delay from onset-to-death, a simple 'naive' calculation that just divides deaths-to-date by cases-to-date will underestimate severity. The `estimate_rolling()` function uses the `estimate_severity()` adjustment to account for delays, and instead compares deaths-to-date with cases-with-known-outcome-to-date.
```